### PR TITLE
Little FlxBackdrop optimization

### PR DESCRIPTION
**Pull request by Sword352**
This little pull request just replace the way the `spacing` FlxPoint is initialized (getting a recycled point from the pool, basically).

### CODE REVIEW

**Comment by MemeHovy** [comment]
nice 👍

File: flixel/addons/display/FlxBackdrop.hx [hunk 1]
```diff
@@ -68,7 +68,7 @@ class FlxBackdrop extends FlxSprite
 	 * Creates an instance of the FlxBackdrop class, used to create infinitely scrolling backgrounds.
 	 *
 	 * @param   graphic     The image you want to use for the backdrop.
-	 * @param   repeatAxes  If the backdrop should repeat on the X axis.
+	 * @param   repeatAxes  The axis where the backdrop should repeat to.
```
Comment (Geokureli) [inline flixel/addons/display/FlxBackdrop.hx:71]:
```suggestion
	 * @param   repeatAxes  The axes on which to repeat. The default, `XY` will tile the entire camera.
```

**Review by Geokureli — CHANGES_REQUESTED** [review]
good catch on that old comment, but now you've awakened the grammar nazi in me

File: flixel/addons/display/FlxBackdrop.hx (hunk 1 above)
Comment (Sword352) [reply flixel/addons/display/FlxBackdrop.hx]: fair, i suck at english lol, thanks

**Review by Geokureli — APPROVED** [review]

**Pull request merged**